src/tools/rustfmt/src/patterns.rs

```diff
@@ -42,6 +42,7 @@ pub(crate) fn is_short_pattern(
 
 fn is_short_pattern_inner(context: &RewriteContext<'_>, pat: &ast::Pat) -> bool {
     match &pat.kind {
+        ast::PatKind::Missing => unreachable!(),
```

Comment:
Could you briefly explain how we know this is unreachable.

Reply:
Gonna assume "no tests were hit" 😆